pkg/client/gen.go

```diff
@@ -373,34 +373,6 @@ func E2EManifest(cfg *GenConfig) *manifest.Manifest {
 	}
 	m.PodSpec.PodSpec.NodeSelector = map[string]string{"kubernetes.io/os": "linux"}
 
-	// Add volume mount, volume, and 3 env vars (for different possible platforms) for SSH capabilities.
-	if len(cfg.SSHKeyPath) > 0 {
```

Comment:
Without using the transforms; this was conditional logic in the e2e manifest itself. Now we can still extract the plugin (without the ssh logic at all) and the transform can act on it at the CLI level of the code and not the gen/plugin lookup state.